### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by NirmalAgnihotri**
### Tracking Info

Resolves #<issue number>

Make sure your branch name conforms to: `<feature|staging|bugfix|...>/<username>/<3-4 word description separated by dashes>`. Otherwise, please rename your branch and create a new PR.

### Changes

What changes did you make?

- Created simple README with steps to start project

### Testing

How did you confirm your changes worked? 

- N/A

### Confirmation of Change 

N/A

### CODE REVIEW

**Review by SamvritSrinath — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+In a separate terminal install dependencies and run the frontend:
+```bash
+$ cd SDCTA
+$ npm install
+$ npm start
+```
+Note: You will need a .env file that defines DATABASE_URL.
+
+(Currently in progress)
```
Comment (vs2961) [inline README.md]: If this is in progress, why not just remove this and put an issue up about it?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+$ npm start
+```
+
+In a separate terminal install dependencies and run the frontend:
+```bash
+$ cd SDCTA
+$ npm install
+$ npm start
+```
+Note: You will need a .env file that defines DATABASE_URL.
```
Comment (vs2961) [inline README.md]: Maybe make a quick reference to the README in the `server/` folder

**Review by vs2961 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (NirmalAgnihotri) [reply README.md:17]: Because its essentially done, going to put the PR up soon.

File: README.md (hunk 2 above)
Comment (NirmalAgnihotri) [reply README.md]: Not exactly sure what you mean by this, do you mean make a comment in a file in the server folder?

File: README.md (hunk 1 above)
Comment (NirmalAgnihotri) [reply README.md]: Removed it, will just add it with that PR

File: README.md (hunk 2 above)
Comment (vs2961) [reply README.md]: There's an existing README in the server/ folder that teaches you how to create a database user and the `.env` file that you need to run the backend

Comment (NirmalAgnihotri) [reply README.md]: Ah, forgot about that, thank you for letting me know. Updated the README

**Review by vs2961 — APPROVED** [review]

**Pull request merged**